### PR TITLE
Implemented get and update translations (task #806)

### PR DESCRIPTION
**Pull request by stepanov**

### CODE REVIEW

File: src/Controller/TranslationsController.php [hunk 1]
```diff
@@ -53,21 +68,53 @@ public function add()
     {
         $translation = $this->Translations->newEntity();
         if ($this->request->is('post')) {
+            debug($this->request->getData());
```
Comment (mamchenkov) [inline src/Controller/TranslationsController.php]: @stepanov please remove `debug()`.

File: src/Controller/TranslationsController.php [hunk 2]
```diff
@@ -53,21 +68,53 @@ public function add()
...
             $translation = $this->Translations->patchEntity($translation, $this->request->getData());
-            if ($this->Translations->save($translation)) {
+            $result = $this->Translations->save($translation);
+            debug($translation);
```
Comment (mamchenkov) [inline src/Controller/TranslationsController.php]: @stepanov please remove `debug()`

File: src/Controller/TranslationsController.php [hunk 3]
```diff
@@ -54,20 +70,48 @@ public function add()
...
+        $languages = $this->Translations->Languages->find('all', ['limit' => 200]);
+        $this->set(compact('translation', 'languages'));
         $this->set('_serialize', ['translation']);
     }
 
     /**
+     *  Add or update method
+     * @return bool     when successfully added or updated returns true, false otherwise
+     */
+    public function addOrUpdate()
```
Comment (georgeconstantinou) [inline src/Controller/TranslationsController.php]: @stepanov from what I understand this method will be accessed through ajax request. You can limit the allowed methods or the request type using the following:
https://book.cakephp.org/3.0/en/controllers/request-response.html#restricting-which-http-method-an-action-accepts
https://book.cakephp.org/3.0/en/controllers/request-response.html#checking-request-conditions

Comment (stepanov) [inline src/Controller/TranslationsController.php]: Added restriction by request method and type

**Pull request merged**